test/e2e/common/runtimeclass.go

```diff
@@ -84,6 +87,165 @@ var _ = ginkgo.Describe("[sig-node] RuntimeClass", func() {
 
 		expectPodRejection(f, e2enode.NewRuntimeClassPod(rcName))
 	})
+
+	/*
+		Release: v1.20
+		Testname: RuntimeClass API
+		Description:
+		The node.k8s.io API group MUST exist in the /apis discovery document.
+		The node.k8s.io/v1 API group/version MUST exist in the /apis/mode.k8s.io discovery document.
+		The runtimeclasses resource MUST exist in the /apis/node.k8s.io/v1 discovery document.
+		The runtimeclasses resource must support create, get, list, watch, update, patch, delete, and deletecollection.
+	*/
+	ginkgo.It(" [NodeConformance] should support RuntimeClasses API operations", func() {
```

Comment:
keeping it as a regular test for now. Also added `[NodeConformance]` so it will run during the PR validation